src/worldcereal/job.py

```diff
@@ -135,6 +139,10 @@ def generate_map(
         # Make a connection to the OpenEO backend
         connection = BACKEND_CONNECTIONS[backend_context.backend]()
 
+    # Get start and end dates for product file naming
+    start = temporal_extent.start_date.replace("-", "")
```

Comment:
Why do the dashes have to go? Is that more readable? These two lines - if we go for the use of `temporal_extent` - are not needed. Can be plugged directly into `filename_prefix` constructor.